hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeysDeleteResponseWithFSO.java

```diff
@@ -44,14 +44,16 @@
 public class OMKeysDeleteResponseWithFSO extends OMKeysDeleteResponse {
 
   private List<OmKeyInfo> dirsList;
+  private long volumeId;
 
   public OMKeysDeleteResponseWithFSO(
       @NotNull OzoneManagerProtocolProtos.OMResponse omResponse,
       @NotNull List<OmKeyInfo> keyDeleteList,
       @NotNull List<OmKeyInfo> dirDeleteList, boolean isRatisEnabled,
-      @NotNull OmBucketInfo omBucketInfo) {
+      @NotNull OmBucketInfo omBucketInfo, long volId) {
```

Comment:
I don't think we will have cases of this class being instantiated with a `null` value assigned to `volId` parameter due to the changes in `getVolumeId` and `getBucketId` methods. Should we annotate it with `@NonNull` as well? (Similarly for the other request classes as well)
```suggestion
      @NotNull OmBucketInfo omBucketInfo, @NotNull long volId) {
```

Reply:
Thanks @JyotinderSingh for the comment. Could you please review the updated patch.